### ISSUE TITLE
UX: Ability to switch addresses easier and show that its switched. 

### ISSUE THREAD

**Issue by patoshii** [bsimic0001/AegisWallet#37]
Currently you can switch addresses, but there is no indicuator that it is switched. If you compare it to mycellium, when an address is switched it will only show that current addresses transactions. In aegis even if you switch an address the transactions still show up from the previous address.

There should be an additional option on the addresses tab where you can select a COLOR that represents that address and have the background color change to that color when that chosen address is switched. This gives it a good UX where you know Green is your personal address and Red is your business address. 

Please check out the mycellium app and try switching between addresses and you can see what I mean. 

In short:
"Switched addresses should only show transactions of that address and have some type of indicator that your using a different address"
